src/services/API/Video.js

```diff
@@ -0,0 +1,20 @@
+import { apiClient } from "@/services/API/RootClient.js";
+
+export default {
+  async createVideo(videoDetails) {
+    // API to create a video object in the DB
+    var response = await apiClient().post(
+      process.env.VUE_APP_BACKEND_VIDEOS,
+      videoDetails
+    );
+    return response.data.id;
+  },
+
+  updateVideo(videoDBId, videoDetails) {
+    // API to update a video object in the DB
+    return apiClient().patch(
```

Comment:
why is this not `.put` instead of `.patch`?
This answer says that `PATCH` is for partial updates: https://stackoverflow.com/questions/31089221/what-is-the-difference-between-put-post-and-patch

Based on the answer, PUT seems more appropriate!